package.json

```diff
@@ -26,14 +26,14 @@
   },
   "engines": {
     "node": ">=6 <15",
```

Comment:
I hadn't noticed this before, but >= node 6 seems like a promise we can't really keep since we only test on node 12 and 14 (and soon 16). So I wonder if node >= 12 wouldn't be a better reflection of reality.

Reply:
Changing the Node.js version would indeed be a better reflection of reality, but I'll update that in the PR that updates our Node.js versions and introduces testing and support for Node.js 16.

It's also worth nothing that we're already not constrained by this promise of Node.js v6 since individual packages in this repository (which are actually installed, and are the actual manifests/contracts) state much higher minimum versions.

Reply:
Done in #713